### PR TITLE
Staged payment locked

### PR DESCRIPTION
**Pull request by agnieszkajarosikloj**
## Description

This PR adds locked state of staged payment.
Screenshot 2022-08-02 at 15 23 51

[Figma Link - Staged payments locked state](https://www.figma.com/file/dCtvv76S8mk5HNApjwmBto/Expenditures?node-id=3802%3A279744)

## TODO

- Add edits in locked state, when [PR](https://github.com/JoinColony/colonyDapp/pull/3609) will be merged.

(Resolves) #3682

### CODE REVIEW

File: src/modules/dashboard/components/ExpenditurePage/Staged/LockedStaged/LockedStaged.css [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+  justify-content: space-between;
+  padding-bottom: 19px;
+}
+
+.milestoneName {
+  padding-bottom: 19px;
+  font-size: var(--size-normal);
+  font-weight: var(--weight-bold);
+  color: var(--dark);
+  letter-spacing: 0.25px;
```
Comment (danbr) [inline src/modules/dashboard/components/ExpenditurePage/Staged/LockedStaged/LockedStaged.css:78]: var(--spacing-normal)

File: src/modules/dashboard/components/ExpenditurePage/Staged/LockedStaged/LockedStaged.css [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+  left: 0;
+  border-radius: 7px;
+  background-color: color-mod(var(--primary) alpha(85%));
+}
+
+.percent {
+  font-size: var(--size-smallish);
+  font-weight: var(--weight-bold);
+  color: color-mod(var(--dark) alpha(85%));
+  letter-spacing: 0.5px;
```
Comment (danbr) [inline src/modules/dashboard/components/ExpenditurePage/Staged/LockedStaged/LockedStaged.css:131]: --spacing-medium

**Review by danbr — APPROVED** [review]
LGTM - well done.

**Comment by willm30** [comment]
@agnieszkajarosikloj I'm getting a few warnings in the console. 

1. When selecting "Staged"
staged-key

2. When submitting the draft and clicking "Stake" on the modal (before locking)
locked-warning2

3. When I click "Lock values"
locked-warnings

Looks good though.

**Review by Julianahlk — COMMENTED** [review]
Very nice! It is working well for me. 

A few comments:
1. In the design, the input field doesn't have a border. However, I think adding a border is a good idea because it makes very clear where to add a name for the milestones. We can double check with @arrenv if we should add it or not and what the color should be.  
Screen Shot 2022-08-05 at 1 09 23 AM
2. I am getting these Popper warnings 
Screen Shot 2022-08-05 at 1 15 44 AM

**Comment by arrenv** [comment]
> 1. In the design, the input field doesn't have a border. However, I think adding a border is a good idea because it makes very clear where to add a name for the milestones. We can double check with @arrenv if we should add it or not and what the color should be.

Just confirming you mean turning these text inputs into typical inputs field designs? At the moment, this would be a scope creep for the project, so, we can revisit it again when we do the wiring up.

**Review by arrenv — CHANGES_REQUESTED** [review]
Looks good @agnieszkajarosikloj, just noticed the font does not match.

I'm pretty sure you're going to include the edit icon in another PR, but I have pointed it out in this screenshot anyway.

8cdba69a-0f27-4164-b1c5-8a6eadc17453

**Review by ArmandoGraterol — APPROVED** [review]
Looks good to me :+1: 

FireShot Capture 695 - Create Expenditure - Colony - wonker - localhost

**Review by arrenv — APPROVED** [review]
Looks good @agnieszkajarosikloj, thank you!

**Pull request merged**